lib/meilisearch/index.rb

```diff
@@ -221,7 +221,10 @@ def delete_all_documents!
     ### SEARCH
 
     # options: A Hash
-    #   show_ranking_score - To see the ranking scores for returned documents
+    # show_ranking_score - To see the ranking scores for returned documents
+    # attributes_to_search_on: Customize attributes to search on at search time.
+    # Available ONLY with Meilisearch v1.3 and newer (optional).
+ 
```

Comment:
Fixed the linter error and use the same layout like for `def delete_documents`
```suggestion
    #   show_ranking_score - To see the ranking scores for returned documents
    #   attributes_to_search_on - Customize attributes to search on at search time.
```